pyartnet/artnet_node.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, host: str, port: int = 0x1936, max_fps: int = 25,
         self._socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)  # UDP
 
         if self.__source_ip is not None and self.__source_ip is not None:
+            self._socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
What does this do?

Reply:
If you attempt to bind the port multiple times it throws an error. This allows it to do it multiple times for example multiple different controllers. I found this to work better than an exception catch as a few instances seemed to have a previous artifact where it wouldn't allow you to bind even after restart. This fixed that issue aswell as the inital problem.